src/App.js

```diff
@@ -364,7 +364,22 @@ export default class App extends Component {
 
     function generateClassCode(allClasses, classesUsed) {
       let code = '';
-      classesUsed.forEach((className) => {
+
+      // Qualifier and PostCartAmountQualifier need to come first since they are inherited by other classes
+      const classNameOrder = ["Campaign", "Qualifier", "PostCartAmountQualifier", "Selector"];
```

Comment:
Because of the new inheritance from `PostCartAmountQualifier`, I needed to make sure the classes were declared before inheriting from them.